packages/core/src/html/NodeProcessor.ts

```diff
@@ -343,8 +395,8 @@ class NodeProcessor {
         const mainHtmlNodes = dom.map((d) => {
           let processed;
           try {
-            processed = this._process(d, context);
-          } catch (err) {
+            processed = this.traverse(d, context);
+          } catch (err: any) {
             err.message += `\nError while rendering '${file}'`;
             logger.error(err);
             processed = createErrorNode(d, err);
```

Comment:
I think you can update this to `import { createErrorNode } from './elements'` since `element` has been converted to typescript. This should also let typescript check the typing of `mainHtmlNodes` which will cause the error.